app/views/govuk_publishing_components/components/docs/cookie_banner.yml

```diff
@@ -0,0 +1,16 @@
+name: Cookie banner
+description: Help users manage their personal data by telling them when you store cookies on their device.
+body: |
+  Setting `data-hide-cookie-banner="true"` on any link inside the banner will overwrite the default action and when clicked will dismiss the cookie banner for a period of 365 days (approx. 1 year).
+accessibility_criteria: |
+  - accept focus
```

Comment:
I think this is missing `Links in the component must...`. 

Also is there anything we can add about the visibility and readability of the cookie banner? I've seen plenty of confusing ones and badly worded language can be a barrier. Something like `text in the banner must be clear and unambiguous`?